terraform/lambda.tf

```diff
@@ -6,21 +6,35 @@ data "archive_file" "lambda_function" { # <1>
 }
 
 resource "aws_s3_bucket" "lambda_sources" { # <2>
-  acl = "private"
-  versioning {
-    enabled = true
+  bucket = "lambda-bucket-2023-demo"
```

Comment:
should we use `bucket_prefix` here?
this would enable arbitrary deployments without collisions in bucket names

Reply:
seems to be fixed